package-lock.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "@etherisc/gif-contracts",
-  "version": "2.0.0-rc.1",
+  "version": "2.0.0-rc.1-0",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@etherisc/gif-contracts",
-      "version": "2.0.0-rc.1",
+      "version": "2.0.0-rc.1-0",
       "license": "Apache-2.0",
       "dependencies": {
         "@chainlink/contracts": "0.4.1",
         "@etherisc/gif-interface": "2.0.0-rc.1-0",
         "@openzeppelin/contracts": "4.7.3"
+      },
```

Comment:
Why did you introduce new dependencies here? this package.json is used to build the npm packages for the gif contracts. we do not want additonal dependencies here that are not required for compiling the contracts.

Reply:
These are only devDependencies? Doesn't work without

Reply:
Openzeppelin has its own `package.json` in the `/contracts` folder and publishes to npm out of this folder. This `package.json` doesn't have any dependencies (or only the ones you want). 
I've added a sample

Reply:
Oh you're right. I completely missed that those are dev dependencies. Just checked, they don't pollute the stack so we're fine. 
I would not switch to a separate package.json in /contracts now. If we want this, we should create a separate ticket and do it in the next release. For now, i would remove the /contracts/package.json again to avoid confusion as all scripts are geared and configured to the one in the root of the project.

Reply:
ok done